client/js/components/user/orgaDataEntry/OrganisationDataForm.vue

```diff
@@ -0,0 +1,355 @@
+          </div>
+        </div>
+
+        <!-- Display Slug and Customer List -->
+        <dl
+          v-if="displaySlug || displayCustomer"
+          class="description-list space-stack-s">
+          <div v-if="displaySlug">
+            <dt class="font-semibold">
+              {{ Translator.trans('organisation.procedurelist.slug') }}
+            </dt>
+            <dd class="color--grey">
+              {{ proceduresDirectlinkPrefix }}/{{ organisation.currentSlugName }}
+            </dd>
+          </div>
+
+          <div v-if="displayCustomer">
+            <dt class="font-semibold">
+              {{ Translator.trans('customer', { count: customers.length }) }}
+            </dt>
+            <dd
+              v-for="(customer, index) in customers"
+              :key="customer.id"
+              class="color--grey inline">
+              {{ customer.name }}<span v-if="index < customers.length - 1">, </span>
+            </dd>
+          </div>
+        </dl>
+      </template>
+
+      <template v-else>
+        <dp-input
+          id="orga_name"
+          v-model="organisation.name"
+          class="mb-2"
+          data-cy="organisationData:name"
+          :name="`${organisation.id}:name`"
+          :label="{
+            text: Translator.trans('name.legal')
+          }"
+          :disabled="!isOrgaDataEditable" />
+
+        <dp-input
+          id="orga_address_street"
+          v-model="organisation.street"
+          class="mb-2"
+          data-cy="organisationData:address:street"
+          :name="`${organisation.id}:address_street`"
+          :label="{
+            text: Translator.trans('street')
+          }"
+          :disabled="!isOrgaDataEditable" />
+
+        <dp-input
+          id="orga_address_postalcode"
+          v-model="organisation.postalcode"
+          data-cy="organisationData:address:postalcode"
+          class="mb-2"
+          :name="`${organisation.id}:address_postalcode`"
+          :label="{
+            text: Translator.trans('postalcode')
+          }"
+          :pattern="isOrgaDataEditable ? '^[0-9]{5}$' : ''"
+          :size="5"
+          :disabled="!isOrgaDataEditable" />
+
+        <dp-input
+          id="orga_address_city"
+          v-model="organisation.city"
+          class="mb-2"
+          data-cy="organisationData:address:city"
+          :name="`${organisation.id}:address_city`"
+          :label="{
+            text: Translator.trans('city')
+          }"
+          :disabled="!isOrgaDataEditable" />
+
+        <dp-input
+          v-if="hasPermission('field_organisation_phone')"
+          id="orga_address_phone"
+          class="mb-2"
+          v-model="organisation.phone"
+          data-cy="organisationData:phone"
+          :name="`${organisation.id}:address_phone`"
+          :label="{
+            text: Translator.trans('phone')
+          }"
+          :disabled="!isOrgaDataEditable" />
+
+        <!-- TODO: show all types instead of disabled select-->
```

Comment:
still a todo?

Reply:
I have to discuss the display of the types with the PO, but for now its done